### ISSUE TITLE
use eocd to search for sigblock

### ISSUE THREAD

**Issue by obfusk** [obfusk/apksigcopier#8]
See #5.

**Comment by mimi89999** [comment]
The comment https://github.com/obfusk/apksigcopier/pull/5#issuecomment-807016776

**Comment by eighthave** [comment]
This part of the _apksigner_ code is not too hard to follow, so it would be worth referring to it:

* https://gitlab.com/eighthave/apksigner/-/blob/9bc2f471c00069db1c08cb76a1caf0f92c3fa762/src/main/java/com/android/apksig/apk/ApkUtilsLite.java#L95
* https://gitlab.com/eighthave/apksigner/-/blob/9bc2f471c00069db1c08cb76a1caf0f92c3fa762/src/main/java/com/android/apksig/SignatureDumper.java#L198
